docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -98,6 +97,10 @@ ninja
 ```
 Refer to the [LVI Mitigation](AdvancedBuildInfo.md#lvi-mitigation) documentation for further information.
 
+Now, using the `ninja install` command will install the SDK in
+`C:/openenclave`. To choose a different location, change
```

Comment:
Is `C:\openenclave` the expected default install path?

Given that the build happens under `C:/Users/test/openenclave/build/x64-Release` and ` -DCMAKE_INSTALL_PREFIX=install` is specified, wouldn't the expected install path be `C:/Users/test/openenclave/build/x64-Release/install`?

Reply:
Thanks for pointing this out. If my suggestion at line #86 is committed, the install path would be c:\openenclave